config/config/wc-config.yaml

```diff
@@ -92,10 +92,6 @@ opa:
   resourceRequests:
     enabled: true
     enforcement: dryrun
-  enforcements:
```

Comment:
I think you can mention [here](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration/v0.8.x-v0.9.x/upgrade-apps.md) that these should be removed  when upgrading.

Reply:
Good point, fixed now :)